File: c#/src/Docmost.Sdk/Models/ExportPageDto.cs
namespace Docmost.Sdk.Models;

public class ExportPageDto
{
    [System.Text.Json.Serialization.JsonPropertyName("pageId")]
    public string PageId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("format")]
    public string Format { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeChildren")]
    public bool? IncludeChildren { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeAttachments")]
    public bool? IncludeAttachments { get; set; }

}

File: go/docmost/models/export_page_dto.go
package models

// ExportPageDto from OpenAPI.
type ExportPageDto struct {
	PageId string `json:"pageId"`
	Format string `json:"format"`
	IncludeChildren *bool `json:"includeChildren,omitempty"`
	IncludeAttachments *bool `json:"includeAttachments,omitempty"`
}
